pallets/crowdfund/src/lib.rs

```diff
@@ -66,18 +66,48 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 		/// Crea un proyecto.
-		pub fn crear_proyecto(origen: OriginFor<T>, nombre: String) -> DispatchResult {
-			// Completar este método.
-			todo!()
+		pub fn crear_proyecto(origen: OriginFor<T>, nombre: Vec<u8>) -> DispatchResult {
+			let _quien = ensure_signed(origen)?;
+		
+			let nombre_acotado: Result<NombreProyecto<T>, _> = NombreProyecto::<T>::try_from(nombre.clone());
+			let nombre_acotado = match nombre_acotado {
+				Ok(v) => v,
+				Err(_) => return Err(Error::<T>::NombreMuyLargo.into()),
+			};
```

Comment:
```suggestion
			let nombre_acotado: NombreProyecto<T> = NombreProyecto::<T>::try_from(nombre.clone()).map_err(|_| Error::<T>::NombreMuyLargo)?;
```

Eso te ahorra el match. Lo que hiciste sigue estando bien.